pandas/core/series.py

```diff
@@ -5004,6 +5004,42 @@ def reindex(  # type: ignore[override]
             tolerance=tolerance,
         )
 
+    @overload
```

Comment:
It seems that the error codes are misaligned, see mypy:

>  pandas/core/series.py:5007: error: Signature of "rename_axis" incompatible with supertype "NDFrame"  [override]
> [...]
> pandas/core/series.py:5008: error: Unused "type: ignore" comment  [unused-ignore]
> pandas/core/series.py:5020: error: Unused "type: ignore" comment  [unused-ignore]
> pandas/core/series.py:5032: error: Unused "type: ignore" comment  [unused-ignore]
> pandas/core/series.py:5044: error: Unused "type: ignore" comment  [unused-ignore]

Reply:
Fixed I think

Reply:
you might want to run mypy (and black) locally :)

> pandas/core/series.py:5007: error: Signature of "rename_axis" incompatible with supertype "NDFrame"  [override]

Reply:
OK understood. Now it seems to be not complaining.